python/graphstorm/gsf.py

```diff
@@ -70,6 +94,10 @@ def initialize(ip_config, backend):
     assert th.cuda.is_available() or backend == "gloo", "Gloo backend required for a CPU setting."
     if ip_config is not None:
         th.distributed.init_process_group(backend=backend)
+        # Use wholegraph for feature fetching if 'wholegraph' folder exists
+        if part_config is not None and os.path.isdir(os.path.join( \
```

Comment:
We can define a use_wholegraph function in gs.util.py. and both dataset.py and gsf.py can import it.

Reply:
Yes! Done.